subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/assignment/internal/KotlinDslAssignment.kt

```diff
@@ -16,11 +16,20 @@
 
 package org.gradle.kotlin.dsl.assignment.internal
 
+import org.gradle.util.internal.IncubationLogger
+
 
 object KotlinDslAssignment {
```

Comment:
:question: couldn't we make this `internal`?

Reply:
`internal` as `internal object KotlinDslAssignment`?

Reply:
It seems we cannot do that for whole class, since it's used also in the `kotlin-dsl` plugin, but we can make emitIncubatingLogMessage internal

Reply:
Let's do it for `emitIncubatingLogMessage` then :+1: